css/footer.css

```diff
@@ -0,0 +1,77 @@
+footer {
+  height: 846px;
+  position: relative;
+}
+
+.f-box-wrapper {
```

Comment:
As highlighted by the last reviewer, there are a few details about styling your form that will make it appear more like the design spec. I think working with the background is a nice place to start.